src/html/layout.html

```diff
@@ -191,6 +193,7 @@
       {%- if favicon %}
       <link rel="shortcut icon" href="{{ pathto('_static/' + favicon, 1) }}"/>
       {%- endif %}
+      <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet">
```

Comment:
Add this to the `css()` macro in https://github.com/SpongePowered/sponge_docs_theme/blob/master/sponge_docs_theme/layout.html#L159. As you can see we already load some fonts from Google so you just need to at Montserrat to the existing link.